bitnami/clickhouse/templates/keeper/headless-service.yaml

```diff
@@ -0,0 +1,39 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.keeper.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "clickhouse.keeper.headlessServiceName" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" (dict "customLabels" .Values.commonLabels "context" .) | nindent 4 }}
+    app.kubernetes.io/component: keeper
+    app.kubernetes.io/part-of: clickhouse
+  {{- if or .Values.keeper.service.headless.annotations .Values.commonAnnotations }}
+  {{- $annotations := include "common.tplvalues.merge" (dict "values" (list .Values.keeper.service.headless.annotations .Values.commonAnnotations) "context" .) }}
+  annotations: {{- include "common.tplvalues.render" (dict "value" $annotations "context" .) | nindent 4 }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  publishNotReadyAddresses: true
+  ports:
+    - name: tcp
+      port: {{ .Values.keeper.containerPorts.tcp }}
+      protocol: TCP
+      targetPort: tcp
+    - name: tcp-raft
+      port: {{ .Values.keeper.containerPorts.raft }}
+      protocol: TCP
+      targetPort: raft
+    {{- if .Values.keeper.service.extraPorts }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.keeper.service.extraPorts "context" .) | nindent 4 }}
+    {{- end }}
```

Comment:
Does it make sense to add the extra ports here? Maybe it could have a different extraPorts section tailored for the headless service

Reply:
done at https://github.com/bitnami/charts/pull/32958/commits/32aab4b048bc1e1c252b740f3aebe56edcd7e0b7